Source/WebCore/css/parser/CSSPropertyParserConsumer+ViewTransition.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+// Copyright (C) 2016-2023 Apple Inc. All rights reserved.
```

Comment:
```suggestion
 * Copyright (C) 2016-2024 Apple Inc. All rights reserved.
```

The view transition stuff was edited recently by Apple folks too :)